react-hyf/server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const path = require('path');
+const BUILD = './client/build';
+
+const PORT = 8000;
+
+const app = express();
+app.use(express.static(path.join(__dirname, BUILD)));
+
+app.get('/', (req, res) => {
+  res.json("I'm working...");
```

Comment:
So the root '/' gives a text response (I'm working...) but should give the index.html as a response. Did you get it working? 

Take another look at the task in the homework; (3) a GET route that sends the index.html file from your public folder in the client as a response to the browser (take the / route).